src/dbnode/storage/limits/query_limits.go

```diff
@@ -89,17 +91,26 @@ func NewQueryLimits(options Options) (QueryLimits, error) {
 		iOpts               = options.InstrumentOptions()
 		docsLimitOpts       = options.DocsLimitOpts()
 		bytesReadLimitOpts  = options.BytesReadLimitOpts()
+		aggDocsLimitOpts    = options.DocsLimitOpts()
 		sourceLoggerBuilder = options.SourceLoggerBuilder()
 
-		docsLimit = newLookbackLimit(
-			iOpts, docsLimitOpts, "docs-matched", sourceLoggerBuilder)
+		additionalDocs = "docs-matched-all"
```

Comment:
Maybe let's name this `all-docs-matched`? Just so they all end in `docs-matched`.